tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "tslint-config-swimlane/base"
+  "extends": "tslint-config-swimlane/base",
+  "rules": {
+    "no-any": false
```

Comment:
🤬

Reply:
// TODO: resolve all the anys

Reply:
https://github.com/swimlane/mongtype/issues/29#issue-810174793